### PR TITLE
Prepare for 2.0.0-rc1 release

### PR DESCRIPTION
**Pull request by yuchaoran2011**
### What changes were proposed in this pull request?
We have a bunch of inconsistent version strings lying around (e.g. 1.3.3, 1.4.0 and 1.3.4-SNAPSHOT). They should all be the same. To prepare for 2.0.0, I replaced all of them with 2.0.0-SNAPSHOT and did testing on the resulting images.

### Why are the changes needed?
Get ready for release

### Does this PR introduce any user-facing change?
Yes. A user-facing bash script has been renamed.

### How was this patch tested?
On GKE by running CRA app.

### CODE REVIEW

File: docs/docs-source/docs/modules/get-started/pages/setup-k8s-cluster.adoc [hunk 1]
```diff
@@ -20,7 +20,7 @@ To create a new cluster, clone the https://github.com/lightbend/cloudflow/[repo]
 NOTE:: This can be done once you have a valid Google account and have access to a project set up for you by the administrator.
 
 * Create a Kubernetes cluster with a specific name on a cloud provider of your choosing. We provide cluster launching scripts for GKE, AKS and EKS. `./create-cluster-[gke|aks|eks].sh <cluster-name>`
-* Make sure your cluster has at least appropriate storage class that offers ReadWriteMany (RWM) capability. Most cloud providers have their preferred storage class for that purpose (e.g. AzureFile on AKS and EFS CSI on EKS). We also provide a script `./install_nfs.sh` that you can run to install NFS that will satisfy the RWM characteristic.
+* Make sure your cluster has at least appropriate storage class that offers ReadWriteMany (RWM) capability. Most cloud providers have their preferred storage class for that purpose (e.g. AzureFile on AKS and EFS CSI on EKS). We also provide a script `./install-nfs.sh` that you can run to install NFS that will satisfy the RWM characteristic.
```
Comment (yuchaoran2011) [inline docs/docs-source/docs/modules/get-started/pages/setup-k8s-cluster.adoc:23]: All our other bash scripts use this convention, so I renamed it to be consistent.

File: release-notes/cloudflow-2.0.0.md [hunk 2]
```diff
@@ -0,0 +1 @@
+# Cloudflow 2.0.0 Release Notes
```
Comment (yuchaoran2011) [inline release-notes/cloudflow-2.0.0.md:1]: To be filled in in a separate PR.

File: kubectl-cloudflow/Makefile [hunk 3]
```diff
@@ -11,9 +11,6 @@ ifeq (release,$(firstword $(MAKECMDGOALS)))
   $(eval $(RELEASE_TAG):;@:) # Make sure arg is not treated as a rule
 endif
 
-
-all: help
```
Comment (yuchaoran2011) [inline kubectl-cloudflow/Makefile:15]: This is removed because it's misleading and repeated. The user can type `make help` to see help info. `make all` shouldn't be printing help messages.

File: integration-tests/itest/go.mod [hunk 4]
```diff
@@ -3,6 +3,6 @@ module github.com/lightbend/cloudflow/integration-test/itest
 go 1.14
 
 require (
-	github.com/onsi/ginkgo v1.12.0
-	github.com/onsi/gomega v1.9.0
+	github.com/onsi/ginkgo v1.12.3
+	github.com/onsi/gomega v1.10.1
```
Comment (yuchaoran2011) [inline integration-tests/itest/go.mod:7]: Looks like doing a `go get` automatically updated these for me. And they seem to be working. So I kept the updated versions.

File: examples/call-record-aggregator/project/cloudflow-plugins.sbt [hunk 5]
```diff
@@ -1,5 +1,5 @@
 // Resolver for the cloudflow-sbt plugin
 //
 resolvers += Resolver.url("cloudflow", url("https://lightbend.bintray.com/cloudflow"))(Resolver.ivyStylePatterns)
-addSbtPlugin("com.lightbend.cloudflow" % "sbt-cloudflow" % "1.3.4-SNAPSHOT")
+addSbtPlugin("com.lightbend.cloudflow" % "sbt-cloudflow" % "2.0.0-rc1")
```
Comment (RayRoestenburg) [inline examples/call-record-aggregator/project/cloudflow-plugins.sbt:4]: AFAIK the release script makes these changes?

File: core/version.sbt [hunk 6]
```diff
@@ -1 +1 @@
-version in ThisBuild := "1.3.4-SNAPSHOT"
+version in ThisBuild := "2.0.0-rc1"
```
Comment (RayRoestenburg) [inline core/version.sbt:1]: I think the release script modifies this?
